scripts/circle-ci/common-containers.yml

```diff
@@ -0,0 +1,17 @@
+cassandra2:
+  image: palantirtechnologies/docker-cassandra-atlasdb:2.2.8
+
+#cassandra3: //Uncomment when Cassanra 3.7 testing is enabled.
+#  image: palantirtechnologies/docker-cassandra-atlasdb:3.7
+
+postgres:
+  image: kiasaki/alpine-postgres
+
+nginx:
+  image: 1science/nginx
+
+socks:
+  image: vimagick/dante:latest
+
+jepson:
```

Comment:
`jepsen` - we don't reference the keys so I guess this is probably fine, but let's try to keep this clear